docs/devsite-help/long-running-operations.md

```diff
@@ -0,0 +1,65 @@
+# Long-running operations
+
+Some API operations can take a relatively long time to complete, such as
+rebooting a virtual machine or performing speech recognition on a large file.
+Here, a "long time" is generally "anything over about 10 seconds". (It may not
+seem a long time in human terms, but it's longer than you want an individual
+RPC to last.)
+
+## The long-running operation pattern in APIs
+
+The *long-running operation* pattern is designed to avoid clients having to
+worry about having too many pending RPCs, or what happens if a network
+connection is dropped while a long-running operation is taking place.
+
+The pattern is simple:
+
+- The client makes a request RPC in the normal way, representing their
+  intent ("reboot this VM" or "recognize text from this audio data").
+- Assuming the request is valid, the server responds with an `Operation`
+  response which contains an operation name.
+- The client can *poll* using that operation name, to determine whether the
+  long-running operation has completed, whether it's perhaps failed, any
+  metadata indicating progress if it's still running, etc.
+
+In many ways, long-running operations are the API equivalent of the
```

Comment:
Not sure if this might be more confusing than enything, and raise questions about where execution happens?

Reply:
Hmm. I personally think it's a helpful model. Let's talk about this one more.